### PR TITLE
Contrib: Add opencensus-contrib-opencensus-proto-util.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-java/issues/1456.

Currently this Util only supports converting Tracing data models. Most of the conversion between `SpanData` and `SpanProto` are identical to [StackdriverV2ExporterHandler](https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/trace/stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandler.java).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 - Add `createWithSender` to `JaegerTraceExporter` to allow use of `HttpSender`
   with extra configurations
 - Add an API `Functions.returnToString()`.
+- Add `opencensus-contrib-opencensus-proto-util` that has helper utilities on converting between
+  Java data models to protos.
```
[the remark this answers is not in the document]
Comment (songy23) [reply CHANGELOG.md]: Yes, that's a typo. Updated.

**Comment by songy23** [comment]
Thanks!

**Comment by songy23** [comment]
> Who else will use this?

Could be other exporters that directly connect to `Collector`, though for now it's only OC-Agent exporter who's using it. I'll move the utils to exporter for the moment.

**Comment by bogdandrutu** [comment]
Perfect, let's not make it an artifact if we don't have to.

**Pull request merged**